les/winnercheque-v1.jsonld

```diff
@@ -1,16 +1,16 @@
 {
-  "@context":
-    {
-		"@version": 1.1,
-		"wc": "https://sphereon-opensource.github.io/vc-contexts/les/winnercheque-v1#",
-		"schema": "http://schema.org/",
+  "@context": {
+    "@version": 1.1,
```

Comment:
should the version be bumped for this?

Reply:
This is a JSON-LD version, not sure what you want to bump there?